packages/block/src/from-rpc.ts

```diff
@@ -42,7 +45,7 @@ export function blockFromRpc(blockParams: any, uncles: any[] = [], options?: Blo
   const header = blockHeaderFromRpc(blockParams, options)
 
   const transactions: TypedTransaction[] = []
-  if (isTruthy(blockParams.transactions)) {
+  if (blockParams.transactions !== undefined) {
```

Comment:
Is it part of the spec that the RPC can return `undefined` for txs? (Not super related to this PR)